src/common/util/Util.hpp

```diff
@@ -1,31 +1,14 @@
 #ifndef Util_hpp
 #define Util_hpp
 
+#include "../../common/util/InputDef.hpp"
 #include "../physics/VectorUtil.hpp"
-#include <SFML/Graphics.hpp>
 
 namespace Util {
+    std::pair<InputDef::ReliableInput, InputDef::UnreliableInput>
+    vectorToNearestMove(sf::Vector2f v);
     sf::Vector2f inputToDirection(bool toggle_up, bool toggle_down, bool toggle_right,
-                                  bool toggle_left, bool toggle_stop) {
-        if (toggle_stop) {
-            return sf::Vector2f(0.f, 0.f);
-        }
-
-        sf::Vector2f direction = sf::Vector2f(0.f, 0.f);
-        if (toggle_up) {
-            direction += sf::Vector2f(0.f, -1.f);
-        }
-        if (toggle_down) {
-            direction += sf::Vector2f(0.f, 1.f);
-        }
-        if (toggle_left) {
-            direction += sf::Vector2f(-1.f, 0.f);
-        }
-        if (toggle_right) {
-            direction += sf::Vector2f(1.f, 0.f);
-        }
-        return VectorUtil::normalize(direction);
-    };
+                                  bool toggle_left, bool toggle_stop);
```

Comment:
Can we move these to `InputUtil.hpp` and delete this file? 
Not a big fan of generalized util files where a bunch of random misc stuff pile up

Reply:
I was surprised you wanted me to add to this file

Reply:
done